services/web/server/src/simcore_service_webserver/users.py

```diff
@@ -36,3 +37,8 @@ def setup_users(app: web.Application):
         strict=True,
     )
     app.router.add_routes(routes)
+
+    # NOTE: scaling web-servers will lead to having multiple tasks upgrading the db
+    # not a huge deal but perhaps this task could be registered as a GC task and
+    # run only when that service is active
```

Comment:
yep, true that.

Reply:
I'd also like only one copy of this task to avoid having unexpected issues

Reply:
OK, i will do a temp solution for this by adding the ``on_cleanup`` to the garbage collector